test/test-server/cli_handler.go

```diff
@@ -0,0 +1,27 @@
+package testserver
+
+import (
+	"encoding/json"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	cliv1 "github.com/confluentinc/ccloud-sdk-go-v2/cli/v1"
+)
+
+// Handler for: /cli/v1/feedbacks
+func handleFeedbacks(t *testing.T) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		var req cliv1.CliV1Feedback
+		err := json.NewDecoder(r.Body).Decode(&req)
```

Comment:
This is a bit more idiomatic. You don't need to use `&` every time you pass it into a function this way.
```suggestion
		req := new(cliv1.CliV1Feedback)
		err := json.NewDecoder(r.Body).Decode(req)
```